src/components/Auth/BackupRecoveryCodesView.tsx

```
@@ -0,0 +1,54 @@
+import React from 'react';
+import { ShieldCheck, Copy, Check, Lock } from 'lucide-react';
+
+interface RecoveryCodesViewProps {
+    backupCodes: string[];
+    onFinish: () => void;
+}
+
+export const BackupRecoveryCodesView: React.FC<RecoveryCodesViewProps> = ({ backupCodes, onFinish }) => {
+    const [copied, setCopied] = React.useState(false);
+
+    const handleCopy = () => {
```

ESLint

[error] Delete `··`

(prettier/prettier)